jwst-masterclass/jupyter_desktop/MANIFEST.in

```diff
@@ -0,0 +1 @@
+graft jupyter_desktop/share
```

Comment:
Rather than include the jupyter-desktop files in our repo, can we just install directly from GitHub?  For example, add this line to the pip section of base-environment.yml:

`git+https://github.com/yuvipanda/jupyter-desktop@d839035`